libr/core/cmd_print.c

```diff
@@ -1613,37 +1625,42 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 				r_print_format (core->print, core->offset,
 					core->block, core->blocksize, name, mode, NULL, NULL);
 			}
+stage_name:
 			free (name);
 		}
 	} else {
-		/* This make sure the structure will be printed entirely */
+		/* Make sure the structure will be printed entirely */
 		const char *fmt = r_str_trim_head_ro (input + 1);
 		int struct_sz = r_print_format_struct_size (core->print, fmt, mode, 0);
 		int size = R_MAX (core->blocksize, struct_sz);
 		ut8 *buf = calloc (1, size);
 		if (!buf) {
-			eprintf ("cannot allocate %d byte(s)\n", size);
+			eprintf ("Cannot allocate %d byte(s) for an entire structure\n", size);
```

Comment:
no need to have an eprintf on every heap allocation

Reply:
I saw quite different approaches throughout this code base. Since we lack Rust's elvis `?` operator in C, what would you consider the most canonical error handling technique for this project? Or perhaps it is already document somewhere but I missed it?